package.json

```diff
@@ -10,7 +10,8 @@
     "test": "npm run flow && npm run jest",
     "coverage": "STORYBOOK_INLINE_APHRODITE=1 jest --coverage",
     "storybook": "start-storybook -p 9001 -c .storybook",
-    "build:storybook": "build-storybook -c .storybook -o docs && git add docs"
+    "styleguidist": "styleguidist server",
+    "build:styleguidist": "styleguidist build"
```

Comment:
Could we add back in the `&& git add docs`? Alternatively you can add a new command like `"git-add-docs": "git add docs",` and then add `"git-add-docs"` to the `"pre-commit"` array below.

Reply:
Ah... I missed that.  Will do.